ext/pcntl/tests/pcntl_rfork_flags.phpt

```diff
@@ -0,0 +1,31 @@
+--TEST--
+Test function pcntl_rfork() with RFCFDG and RFFDG flags.
+--SKIPIF--
+<?php
+	if (!extension_loaded('pcntl')) die('skip pcntl extension not available');
+	elseif (!extension_loaded('posix')) die('skip posix extension not available');
+  if (!function_exists('pcntl_rfork')) die('skip pcntl_rfork unavailable');
+?>
+--FILE--
+<?php
+echo "\n*** Test with RFFDG and RFCFDG flags ***\n";
+$pid = pcntl_rfork(RFFDG);
+if ($pid > 0) {
+	pcntl_wait($status);
+  var_dump($pid);
+} else {
+	var_dump($pid);
+  exit;
+}
+
+$pid = pcntl_rfork(RFCFDG);
+if ($pid > 0) {
+  var_dump($pid);
+  sleep(2);
```

Comment:
Could you document the reason for using sleep instead of pcntl_wait to check if a child process exited? The latter seems like the test would be faster

Reply:
worse than that :-) bad copy paste while moving around test cases in various files.